### PR TITLE
Spelling Fixes

### PR DESCRIPTION
**Pull request by davis-matthew**
Some spelling fixes for flang1exe

### CODE REVIEW

File: tools/flang1/flang1exe/comm.c [hunk 1]
```diff
@@ -3449,7 +3449,7 @@ construct_list_for_pure(int arg, int mask, int list)
 
 /* This will find temp_reference for pure communication.
  *  lhs=a(i,j,2), arg=b(2,i,:) will be tmp=tmp(i,j,:)
- * heading dimension from lhs, talling from arg.
+ * heading dimension from lhs, tailling from arg.
```
Comment (schweitzpgi) [inline tools/flang1/flang1exe/comm.c]: I believe this should be "trailing".

Comment (davis-matthew) [reply tools/flang1/flang1exe/comm.c]: Yes, I believe you are right. I have updated the request to include this change.

**Comment by davis-matthew** [comment]
I apologize, I was doing a personal test and was on the wrong branch. I have reverted the pull request branch back to where it was previously. Thanks for letting me know.

**Comment by kiranchandramohan** [comment]
I am looking for two +2s to merge PR 875. https://github.com/flang-compiler/flang/pull/875
This PR consists of some text comment changes (spelling correction) and also one function name change (also a spelling correction).
I tested on Aarch64 and looks fine.

Plan to use the squash and merge feature to merge.

**Comment by kiranchandramohan** [comment]
Considering this PR based on the PR merge process (see link below) written by @gklimowicz.
http://lists.llvm.org/pipermail/flang-dev/2020-June/000390.html

-> This patch is up to date with master and does not need a rebase. 
->  I am owning this PR for the merge purpose.
-> This patch was already approved by @schweitzpgi.
-> Further tested on Aarch64 by myself, AMD64/X86_64 by @shivaramaarao and Power by @gklimowicz.

Based on the above facts which satisfies the PR merge process, I am merging this PR.

**Comment by sscalpone** [comment]
Woo hoo!

**Comment by gklimowicz** [comment]
On Jul 7, 2020, at 12:57 PM, Steve Scalpone <notifications@github.com<mailto:notifications@github.com>> wrote:


Woo hoo!

I'll second that!

**Pull request merged**